ipam/ring/ring_test.go

```diff
@@ -506,6 +506,22 @@ func TestOwner(t *testing.T) {
 
 }
 
+func makePeerName(i int) mesh.PeerName {
+	peer, _ := mesh.PeerNameFromString(fmt.Sprintf("%02d:%02d:00:00:00:00", i/10, i%10))
```

Comment:
Although we don't have tests that use more than 100 peers at the same time, `TestFuzzRingHard` does add and remove peers while incrementing `nextPeerID`, with the effect that the number seen by `makePeerName` goes over 100.